File: blog/src/main/java/com/slmanju/blog/ControlFlow.java
package com.slmanju.blog;

public class ControlFlow {

    public static void main(String[] args) {
        int a = 10;
        int b = 3;
        findMax(a, b);
        findMax2(a, b);
    }

    private static void findMax2(int a, int b) {
        String result = (a > b) ?  a + " is greater than " + b : b + "is greater than " + a;
        System.out.println(result);
    }

    private static void findMax(int a, int b) {
        int max;
        if (a > b) {
            max = a;
        } else {
            max = b;
        }
        System.out.println("Max value is " + max);
    }

}

File: blog/src/main/kotlin/com/slmanju/blog/ControlFlow.kt
package com.slmanju.blog

fun main(args: Array<String>) {
    val a = 10
    val b = 3
    findMax(a, b)
    findMax2(a, b)

    println(directReturn(35))
}

private fun directReturn(age: Int): String {
    return if (age < 21) "You are a kid" else "You are an adult"
}

private fun findMax2(a: Int, b: Int) {
    val result = if (a > b) "$a is greater than $b" else " $b is greater than $a"
    println(result)
}

private fun findMax(a: Int, b: Int) {
    val max: Int = if (a > b) {
        a
    } else {
        b
    }
    println("Max value is $max")
}